src/libumockdev-preload.c

```diff
@@ -1516,6 +1516,12 @@ ssize_t readlinkat(int dirfd, const char *pathname, char *buf, size_t bufsiz)
     return r;
 }
 
+/* A readlinkat fortify wrapper that is used when -D_FORTIFY_SOURCE is used. */
+ssize_t __readlinkat_chk(int dirfd, const char *pathname, char *buf, size_t bufsiz, size_t buflen)
```

Comment:
I guess we need the same treatment here as for `__realpath_chk` and add a definition.

```suggestion
ssize_t __readlinkat_chk(int dirfd, const char *pathname, char *buf, size_t bufsiz, size_t buflen);
ssize_t __readlinkat_chk(int dirfd, const char *pathname, char *buf, size_t bufsiz, size_t buflen)
```